tests/topotests/zebra_ipv6_import_table/r1/import_mrib_table_3.json

```diff
@@ -0,0 +1,255 @@
+{
+    "2001:db8:db02::/48": [
+        {
+            "prefix": "2001:db8:db02::/48",
+            "prefixLen": 48,
+            "protocol": "connected",
+            "vrfId": 0,
```

Comment:
I would remove vrfID, internalStatus and internalFlags from the json here.  These are not 100% guaranteed to stay the same and frankly we don't want to have to fix this in the future.